client/src/lib/components/zzz/ZipZapZop.svelte

```diff
@@ -1,18 +1,26 @@
 <script lang="ts">
+    import {
+        DndContext,
+        type DragEndEvent,
+        DragOverlay,
+        type DragStartEvent,
+        type UniqueIdentifier,
+    } from '@dnd-kit-svelte/core';
+    import ActionButton from './ActionButton.svelte';
     import Deadline from './Deadline.svelte';
+    import Draggable from './Draggable.svelte';
+    import Droppable from './Droppable.svelte';
     import { PlayerAction } from '$lib/models/game';
     import type { State } from '$lib/zzz/state.svelte';
+    import { assert } from '$lib/utils/assert';
 
     interface Props {
         zzz: State;
     }
 
     const { zzz }: Props = $props();
-
-    let mousePosition = $state({ x: 0, y: 0 });
-    let dragStartPos = $state({ x: 0, y: 0 });
-    let nextAction: PlayerAction | null = $state(null);
-    let hoveredPlayer: number | null = $state(null);
+    let targetedPlayer: UniqueIdentifier | null = $state<UniqueIdentifier | null>(null);
+    let draggedButton: UniqueIdentifier | null = $state<UniqueIdentifier | null>(null);
```

Comment:
The type annotation is redundant here.
```suggestion
    let targetedPlayer = $state<UniqueIdentifier | null>(null);
    let draggedButton = $state<UniqueIdentifier | null>(null);
```